Soccar [UI]/src/nl/socnet/message/SwitchTeamMessage.java

```diff
@@ -0,0 +1,42 @@
+package nl.socnet.message;
+
+import nl.soccar.library.enumeration.TeamColour;
+import nl.soccar.socnet.message.Message;
+import nl.soccar.socnet.message.MessageConstants;
+import nl.soccar.socnet.message.MessageEvent;
+import nl.socnet.message.handler.SwitchTeamMessageHandler;
+
+/**
+ *
+ * @author PTS34A
+ */
+@MessageEvent(id = MessageConstants.SWITCH_TEAM_MESSAGE_ID, handler = SwitchTeamMessageHandler.class)
+public final class SwitchTeamMessage extends Message {
+
+    private final String username;
+    private final TeamColour team;
+
+    public SwitchTeamMessage(String username, TeamColour team) {
+        this.username = username;
+        this.team = team;
+    }
+
+    public SwitchTeamMessage() {

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)